.github/workflows/sub-deploy-integration-tests-gcp.yml

```diff
@@ -473,8 +304,8 @@ jobs:
       # VM and to the container might require more steps in this workflow, and additional
       # considerations.
       #
-      # The disk mounted in the VM is located at /dev/$DISK_NAME, we want the root `/` of this disk to be
-      # available in the docker container at two different paths:
+      # The disk mounted in the VM is located at /dev/sdb, we mount the root `/` of this disk to the docker
+      # container, and might have two different paths (if lightwalletd state is needed):
```

Comment:
Nitpick: This change seems to be a mistake in the merge, the old docs were correct.

Reply:
Fixed with: [fix(ci): use better comment](https://github.com/ZcashFoundation/zebra/pull/7682/commits/a32b3742ebf75221d3a60d7a1c033c7fe3e0a789)